### DISCUSSION TITLE
Minor version updates?

### DISCUSSION THREAD

**Discussion by andyburgessmd** [Budibase/budibase#13673] [General]
Seen the minor version number updates a bit in the last week (2.24 -> 2.25 - 2.26) with very little patches and nothing obvious to show. Should we expect a big obvious change soon?

**Comment by mike12345567** [comment]
Hi @andyburgessmd - Budibase used to release features on a monthly basis, with bug fix patches throughout the month - however a few months ago we transitioned to a different release model, allowing more regular feature releases.

The version increments are as follows:
1. Patch, 0.0.X - patch updates are bug fixes/very minor changes, these shouldn't be noticeable at all.
2. Minor, 0.X.0 - minor updates a feature releases, we'll usually update the [changelog](https://docs.budibase.com/changelog) with information about these around the release. The rate at which these will happen is not easy to define as they release as teams complete the features, therefore there could be a few in a row. For example:
    - 2.24 replaced our old date picker component with a completely new implementation
    - 2.25 included new attachment support in automation row actions
    - 2.26 includes a new user column type which improves the binding experience (a single user column type, the old user column type is always an array when used in bindings). 
3. Major, X.0.0 - major version increments will include very big updates to the platform, which may require some updates to apps before upgrading, e.g. a breaking change. For example this could be a major change to infrastructure required for self hosting, with some new features introduced alongside this.
